Mage.Sets/src/mage/cards/j/JayaBallard.java

```diff
@@ -0,0 +1,237 @@
+import mage.abilities.Ability;
+import mage.abilities.LoyaltyAbility;
+import mage.abilities.common.PlanswalkerEntersWithLoyalityCountersAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.AsThoughEffectImpl;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.ReplacementEffectImpl;
+import mage.abilities.effects.common.AddConditionalManaEffect;
+import mage.abilities.effects.common.GetEmblemEffect;
+import mage.abilities.effects.common.discard.DiscardUpToNDrawThatManySourceEffect;
+import mage.abilities.mana.conditional.ConditionalSpellManaBuilder;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.AsThoughEffectType;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Layer;
+import mage.constants.Outcome;
+import mage.constants.SubLayer;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.filter.common.FilterInstantOrSorcerySpell;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.CardTypePredicate;
+import mage.game.Game;
+import mage.game.command.Emblem;
+import mage.game.events.GameEvent;
+import mage.game.events.ZoneChangeEvent;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author Will
+ */
+public class JayaBallard extends CardImpl {
+
+    public JayaBallard(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.PLANESWALKER}, "{2}{R}{R}{R}");
+
+        addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.JAYA);
+
+        this.addAbility(new PlanswalkerEntersWithLoyalityCountersAbility(5));
+
+        // +1: Add {R}{R}{R}. Spend this mana only to cast instant or sorcery spells.
+        this.addAbility(
+            new LoyaltyAbility(
+                new AddConditionalManaEffect(
+                    Mana.RedMana(3),
+                    new ConditionalSpellManaBuilder(
+                        new FilterInstantOrSorcerySpell())),
+                1));
+
+        // +1: Discard up to three cards, then draw that many cards.
+        this.addAbility(
+            new LoyaltyAbility(
+                new DiscardUpToNDrawThatManySourceEffect(3),
+                1));
+        // −8: You get an emblem with "You may cast instant and sorcery cards from your graveyard. If a card cast this way would be put into your graveyard, exile it instead."
+        this.addAbility(
+            new LoyaltyAbility(
+                new GetEmblemEffect(new JayaBallardEmblem()),
+                -8));
+    }
+
+    public JayaBallard(final JayaBallard card) {
+        super(card);
+    }
+
+    @Override
+    public JayaBallard copy() {
+        return new JayaBallard(this);
+    }
+}
+
+class JayaBallardEmblem extends Emblem {
```

Comment:
Emblems should be located in mage.game.command.emblems.
I guess there was some handle image background for this.